### ISSUE TITLE
wxAuiToolbar docking

### ISSUE THREAD

**Issue by tomkulaga** [nhold/wxWidgets#126]
When i grab an undocked wxAui toolbar by the gripper and try to dock it into its place, it snaps instantly. If i grab it by the "window title", it shows a hint like a dockpane and not like a toolbar, then proceeds to dock in.

Is this intended?

**Comment by mjmacleod** [comment]
No I don't think so, sounds like a bug to me.
Current sample probably needs some toolbars added to help test with this.

**Comment by Kinaou** [comment]
This behavior is also encountered in the 'old aui', so it is not specific to the 'new aui'.

Furthermore, I am not shocked by this behavior. In the first case, the gripper moves the toolbar directly in its environment whereas in the second case, the toolbar is embedded in the a floating window and behaves as is.

**Comment by mjmacleod** [comment]
Agreed, I'd be surprised to find any 'branch specific' toolbar bugs the changes have nothing really to do with the toolbars.
That said it would be nice if the new sample incorporated toolbars again, so that testing for toolbar specific stuff is at least possible.

**Comment by dronesalot** [comment]
![toolbar1](https://cloud.githubusercontent.com/assets/9636311/12145811/5ea9c326-b45e-11e5-8cee-0e313f99e9ca.jpg)
![toolbar2](https://cloud.githubusercontent.com/assets/9636311/12145812/5ea9dd16-b45e-11e5-9e19-ef6b8ba72cf5.jpg)

I also have had toolbar issues for years with AUI. I've been meaning to fix it for a while. If you can see in my screenshots dragging the docked toolbar moves a shadow around the pane.
